spec/models/article_spec.rb

```diff
@@ -206,7 +206,7 @@ def build_and_validate_article(*args)
     end
 
     describe "liquid tags" do
-      xit "is not valid if it contains invalid liquid tags" do
+      it "is not valid if it contains invalid liquid tags" do
```

Comment:
@benhalpern I'm very unclear about this test. This line changed in https://github.com/forem/forem/pull/10717, but it isn't clear why. The test passes now, as is, so I thought I'd remove the `x` anyway, but I'm not sure why it was `x` in the first place?

Reply:
I'm also not sure it was `x` in the first place, but it could very well have been a flakiness issue?

The test seems appropriate in that it is describing this case:

<img width="383" alt="Screen Shot 2022-12-02 at 12 14 33 PM" src="https://user-images.githubusercontent.com/3102842/205348337-8b6847af-5de4-40cd-a5f3-a0183a44d1f7.png">

Whatever my reasoning was, it seems pretty hacky, so proceed as you feel most appropriate, given that.